grails-app/conf/plugin.groovy

```diff
@@ -162,3 +162,23 @@ mapdownloads {
     }
 }
 
+hibernate {
+    dialect =  'org.hibernate.dialect.H2Dialect'
+    cache {
+        queries= false
+        use_second_level_cache= false
+        use_query_cache= false
+    }
+}
+
+database {
+    directory = '/data/ala-hub/database'
```

Comment:
A file-based database will not work with our HA load balancer, so this is a non-starter for production biocache-hubs. Simon is aware of this issue, but unsure about why it is hardoded here?